### ISSUE TITLE
tip not connected to PE

### ISSUE THREAD

**Issue by arekm** [lpcvoid/note-safer-t12#1]
Tip should also get PE connection. There seem to even be PE marking on front board, unconnected.

**Comment by lpcvoid** [comment]
Good point. I currently don't do all that much electronics work, but I'll update this when I break out the iron the next time.
